tests/index.test.js

```diff
@@ -377,3 +377,21 @@ test('microsoft stream links returns undefined id if id missing', (t) => {
   t.is(obj.id, null);
   t.is(obj.service, 'microsoftstream');
 });
+
+/**
+ *  TikTok should be able to find these patterns:
+ *
+ *  Urls:
+ *  https://www.tiktok.com/@brickabrackcommunity/video/6950630446614990085
+ *  https://www.tiktok.com/@brickabrackcommunity/video/6950630446614990085?lang=fr&is_copy_url=1&is_from_webapp=v1
+ *  https://www.tiktok.com/@brickabrackcommunity/video/6950630446614990085?sender_device=pc&sender_web_id=6930592755036374533&is_from_webapp=v1&is_copy_url=0
+ *
+ *  Not supported yet (requires a fetch):
```

Comment:
We should probably add a note about this to the Readme in the `Tiktok` patterns section, to make it more visible which Tiktok patterns this module doesn't support.

Reply:
I added a note about the unsupported TikTok URL to the Readme via https://github.com/radiovisual/get-video-id/pull/67/commits/6d1a888239a1476d3db86251bbfd5e13c3dd1267.